docs/pages/deploy-a-cluster/aws-gslb-proxy-peering-ha-deployment.mdx

```diff
@@ -0,0 +1,282 @@
+in the Route53 GSLB architecure using [VPC Peering](https://docs.aws.amazon.com/vpc/latest/peering/what-is-vpc-peering.html) 
+or [Transit Gateways](https://docs.aws.amazon.com/vpc/latest/tgw/what-is-transit-gateway.html).
+</Admonition>
+
+Internal NLB Auth Service ports
+
+| Port | Description |
+| - | - |
+| `3025` | TLS port used by the Auth Service to serve its API to other Nodes in a cluster |
+
+## Cluster state backend
+
+The Teleport Auth Service stores cluster state (such as dynamic configuration
+resources) and audit events as key/value pairs. In high-availability
+deployments, you must configure the Auth Service to manage this data in a
+key-value store that runs outside of your cluster of Teleport instances.
+
+The Teleport Auth Service manages the creation of any required DynamoDB tables itself, 
+and does not require them to exist in advance.
+
+<Admonition title="Required permissions">
+
+The Auth Service instances needs [permissions to read from and write to DynamoDB](https://goteleport.com/docs/reference/backends/#dynamodb), 
+as well as to create tables.
+
+</Admonition>
+
+## Session recording backend
+
+Highly available Teleport deployments use an object storage service for
+persisting session recordings.
+
+Depeding on how you choose to deploy the Teleport cluster, the Teleport Auth Service can create the required S3 bucket if it doens't already exist.
+
+<Admonition title="Required permissions">
+
+The Auth Service instances [need permissions to get S3 buckets](https://goteleport.com/docs/reference/backends/#s3) as well as to create, get, list, 
+and update objects.
+
+</Admonition>
+
+## TLS credential provisioning 
+
+High-availability Teleport deployments require a  system to fetch TLS
+credentials from a certificate authority like Let's Encrypt, AWS Certificate
+Manager, Digicert, or a trusted internal authority. The system must then
+provision Teleport Proxy Service instances with these credentials and renew them
+periodically. 
+
+For high-availability deployments that use Let's Encrypt to supply TLS
+credentials to Teleport instances running behind a load balancer, you need
+to use the [ACME
+DNS-01](https://letsencrypt.org/docs/challenge-types/#dns-01-challenge)
+challenge to demonstrate domain name ownership to Let's Encrypt. In this
+challenge, your TLS credential provisioning system creates a DNS TXT record with
+a value expected by Let's Encrypt.
+
+Each Teleport Proxy Service instance expects TLS credentials for HTTPS to be available at the file
+paths `/etc/teleport-tls/tls.key` (private key) and `/etc/teleport-tls/tls.crt`
+(certificate).
+
+## Global Server Load Balancing with Route 53
+
+[Latency-based routing](https://docs.aws.amazon.com/Route53/latest/DeveloperGuide/routing-policy-latency.html) 
+in a private hosted zone must be used to ensure traffic from Teleport 
+resources are routed to the closest or lowest latency path Proxy NLB based on the region of 
+the VPC the resource is connecting from.
```

Comment:
Does this assume the resources being connected are also hosted on AWS? How do users resolve DNS if its a private hosted zone?

Reply:
That is a mistake. I'll update it to reflect that the zone should be public.

Reply:
It's probably worth mentioning that route53 latency-based routing is only optimized to route traffic to AWS data centers so any resource outside AWS could have varied routing performance compared to the AWS resources.

Reply:
As long as route53 is routing to NLBs as this guide suggest it shouldn't be an issue.